src/components/Game.js

```diff
@@ -157,6 +157,12 @@ function Game({
     }))
   );
 
+  //Orientation
+  window.onorientationchange = function (event) {
```

Comment:
Good idea. However, you shouldn't modify the `window` object's properties directly, as what happens when a user navigates away from the page to the lobby, or some other page? The component would be unmounted, but this `onorientationchange` listener would still be present, so it would have a stale reference to call `playLayout` and `setCardOrientation`, causing errors.

To attach a global event listener in a component soundly, you should use the `useEffect()` hook along with the `addEventListener()` and `removeEventListener()` functions. See  `useKeydown.js` for an example of this.

Reply:
> Please also rebase this branch on the latest `develop`! It seems like you have an old version of `User.js`.

@ekzhang 
I did `git fetch upstream` but It doesnt update my forked repo. Can you help?

Reply:
> Please also rebase this branch on the latest `develop`! It seems like you have an old version of `User.js`.

@ekzhang Hmm, Can you help me in rebasing?
I tried `git fetch` & `git pull` no its says its up-to-date.
I face this problem many times when I contribute to different issues on the same project you will see the trace/commit history of all I have done in any PR. 
Any advice?

Reply:
@cynthiakedu Can you help with rebasing?